src/API/API.js

```diff
@@ -8,6 +8,16 @@ const api = {
     return data.movies
   },
 
+  getAMovie: async id => {
```

Comment:
Attaboy. One small step for man, one giant step to grab the remote and watch and rate this movie.